Raven.Studio.Html5/App/commands/database/documents/getCollectionsLabelsCommand.ts

```diff
@@ -0,0 +1,59 @@
+﻿import commandBase = require("commands/commandBase");
+import database = require("models/resources/database");
+import collection = require("models/database/documents/collection");
+import queryUtil = require("common/queryUtil");
+
+class getCollectionsLabelsCommand extends commandBase {
+
+    /**
+    * @param ownerDb The database the collections will belong to.
+    */
+    constructor(private collections: collection[], private ownerDb: database) {
+        super();
+
+        if (!this.ownerDb) {
+            throw new Error("Must specify a database.");
+        }
+    }
+
+
+    execute(): JQueryPromise<collection[]> {
+        var task = $.Deferred();
+
+        var requests = this.collections.map(collection => {
+            return {
+                Url: "/queries/",
+                Headers: {},
+                Query: "?id=" + queryUtil.escapeTerm("Raven/StudioConfig")
+            }
+        });
+        this.post("/multi_get?parallel=yes", JSON.stringify(requests), this.ownerDb, null, 0)
```

Comment:
If you have 10 collections you performs 10 the same requests. It is a big waste.